suite/spdz/src/main/java/dk/alexandra/fresco/suite/spdz/datatypes/SpdzInputMask.java

```diff
@@ -7,37 +7,37 @@
  * An inputmask for player_i is random value r 
  * shared among parties so that only player_i knows the real value r.
  */
-public class SpdzInputMask implements Serializable{
+public class SpdzInputMask implements Serializable {
```

Comment:
Why is it serializable?

Reply:
There are actually a few classes that are Serializable - is that needed?

Reply:
Yes, when using the file based streamed storage to pull the preprocessed data from disk, the storage requires objects to extend Serializable (to be used by the ObjectInputStream).

Reply:
The dk.alexandra.fresco.framework.sce.resources.storage.Storage class uses the Serializable interface to store objects. It can probably be changed, but requires some changes in core

Reply:
Ok